### PR TITLE
add -o OUTPUT_FILE

### PR DESCRIPTION
**Pull request by dhcdht**

### CODE REVIEW

File: lib/objc_dependency_tree_generator.rb [hunk 1]
```diff
@@ -136,8 +139,12 @@ def dependencies_to_s
       s = s + json_result.to_json if @options[:output_format] == 'json' || @options[:output_format] == 'json-var'
       s = s + json_result.to_yaml if @options[:output_format] == 'yaml'
     end
+    if @options[:target_file_name]
+        target = File.open(@options[:target_file_name], 'w')
+        target.write("#{s}")
+    end
     s
```
Comment (PaulTaykalo) [inline lib/objc_dependency_tree_generator.rb]: I would suggest not to show output to stdout if output file is specified

Comment (PaulTaykalo) [inline lib/objc_dependency_tree_generator.rb]: @dhcdht

Comment (dhcdht) [inline lib/objc_dependency_tree_generator.rb]: ok

**Comment by PaulTaykalo** [comment]
Thanks, @dhcdht

**Pull request merged**